kotsadm/pkg/policy/getters.go

```diff
@@ -0,0 +1,52 @@
+package policy
+
+import (
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/kots/kotsadm/pkg/store"
+)
+
+func appSlugFromAppIDGetter(vars map[string]string) (map[string]string, error) {
+	if appSlug, _ := vars["appSlug"]; appSlug != "" {
+		return nil, nil
```

Comment:
should this return an empty map?  i don't know if this will panic during template evaluation

Reply:
TODO: fix this